### PR TITLE
Fixing typos/syntax errors

### PR DESCRIPTION
**Pull request by zvr**
This fixes various issues that the (new) spec-parser points out.

Among them:
- mismatches between "name" property and file name and title
- vocabularies entries not in lowerCamelCase
- missing sections ("Summary" in most/all cases)
- empty sections ("Properties" in most/all cases)
- ... and probably some others I don't remember

### CODE REVIEW

File: model/Security/Vocabularies/CvssSeverityType.md
Comment (goneall) [inline model/Security/Vocabularies/CvssSeverityType.md:1]: This change is consistent with the naming conventions of SPDX, however, the security profile is attempting to have the resultant JSON Schema match the CVSS schema and this change will make them different.

Since there is a direct mapping, I'm OK with using the SPDX conventions, but @jeff-schutt @tsteenbe @rnjudge should also review

I would move this into a separate PR to give the security team time to review.  If the current state causes the spec-parser to fail, then I would suggest merging it in and adding a new issue to discuss the incompatibility.

**Review by goneall — CHANGES_REQUESTED** [review]
See comment - suggest moving one of the changes to a separate PR unless it causes the spec-parser to fail.

File: model/Security/Vocabularies/CvssSeverityType.md
Comment (zvr) [reply model/Security/Vocabularies/CvssSeverityType.md:1]: This is about entries for a vocabulary, which are automatically converted to IRIs of individuals in RDF.

The process of generating the JSON Schema already does many transformations, as it's not a 1:1 correspondence. If the requirement is to have them appear as upper-case strings, this is easily done at that stage.

Comment (goneall) [reply model/Security/Vocabularies/CvssSeverityType.md:1]: > The process of generating the JSON Schema already does many transformations, as it's not a 1:1 correspondence. If the requirement is to have them appear as upper-case strings, this is easily done at that stage.

Based on the above, it sounds like we can go ahead and move forward with this PR pending one more review.

I'll leave it up to the security profile group to decide if it is important to maintain uppercase.

**Review by goneall — APPROVED** [review]

File: model/Security/Vocabularies/CvssSeverityType.md
Comment (kestewart) [reply model/Security/Vocabularies/CvssSeverityType.md:1]: ok, let's apply this, and then ask them to do a pass.

**Review by kestewart — APPROVED** [review]
Apply these changes, but then let's get the security team to review.

They'll need to update their examples.

**Pull request merged**
